packages/cron/src/lib/cid.js

```diff
@@ -0,0 +1,24 @@
+import { CID } from 'multiformats/cid'
+import { InvalidCidError } from './errors.js'
+
+/**
+ * Parse CID and return normalized b32 v1
+ *
+ * @param {string} cid
+ */
+export function normalizeCid (cid) {
+  try {
+    const c = CID.parse(cid)
+    return c.toV1().toString()
+  } catch (err) {
+    throw new InvalidCidError(cid)
+  }
+}
+
+export function denormalizeCid (cid) {
+  try {
+    return CID.parse(cid).toV0().toString()
+  } catch (err) {
+    throw new InvalidCidError(cid)
+  }
```

Comment:
It's not an invalid CID if it can't be downgraded. We should let this throw if the CID cannot be downgraded to v0.

```suggestion
  try {
    cid = CID.parse(cid)
  } catch (err) {
    throw new InvalidCidError(cid)
  }
  return cid.toV0().toString()
```